### PR TITLE
[Merged by Bors] - feat(RingTheory/Unramified): Classification of unramifield field extensions.

### PR DESCRIPTION
**Pull request by erdOne**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by Ruben-VandeVelde — APPROVED** [review]
maintainer merge

File: Mathlib/Algebra/Polynomial/Taylor.lean
Comment (riccardobrasca) [inline Mathlib/Algebra/Polynomial/Taylor.lean:128]: `y` can be implicit here.

File: Mathlib/RingTheory/Unramified/Field.lean
Comment (riccardobrasca) [inline Mathlib/RingTheory/Unramified/Field.lean:122]: The name is strange, isn't it? There is no `field` assumption.

**Review by riccardobrasca — APPROVED** [review]
Thanks!

bors d+

File: Mathlib/RingTheory/Unramified/Field.lean
Comment (erdOne) [reply Mathlib/RingTheory/Unramified/Field.lean:122]: The statement is "an unramified algebra over a field is reduced", so I gave it the name `FormallyUnramified.isReduced_of_field`. Do you have a better suggestion?

File: Mathlib/Algebra/Polynomial/Taylor.lean
Comment (erdOne) [reply Mathlib/Algebra/Polynomial/Taylor.lean:128]: It needs to be explicit in my application. Do you prefer making `y` implicit here and using `(y := _)` at the application?

Comment (riccardobrasca) [reply Mathlib/Algebra/Polynomial/Taylor.lean:128]: If you need explicit then no problem leaving as it is. Thanks!

File: Mathlib/RingTheory/Unramified/Field.lean
Comment (riccardobrasca) [reply Mathlib/RingTheory/Unramified/Field.lean:122]: Well, not really. Let's merge this, we can always change the name later.

**Comment by grunweg** [comment]
Coming here from PR triage: is there anything left to do on this PR? It seems this is just waiting for somebody to press the "merge" button.

**Comment by urkud** [comment]
I merged `master` to see if CI is still happy. @riccardobrasca Could you please have another look?

**Comment by riccardobrasca** [comment]
LGTM, but there is still a build error.

**Comment by erdOne** [comment]
bors merge

**Pull request closed without merging**
